app/models/solr_document.rb

```diff
@@ -150,9 +150,35 @@ def extents
   end
 
   def fetch_html_safe(field)
+    return fetch_summary_note(field) if field == "summary_storage_note_ssm"
     fetch(field, []).map(&:html_safe)
   end
 
+  def fetch_summary_note(field)
+    note = fetch(field, [])
+    return note.map(&:html_safe) if note == [] || !note[0].match(/Firestone Library \(mss\): Boxes (?:[A-Z]-\d{6}; )+/)
```

Comment:
We talked about using a named capture group more like `/^(.*: Boxes )(?:[A-Z]-\d{6}; )+/` so we don't have to hard code the library name.

Reply:
For longer postfix conditionals like this it's nice to put them in a small private method and give them a name for readability.